pkg/operator/operator.go

```diff
@@ -105,16 +104,49 @@ func NewOperator(ctx context.Context, opts Options) (Operator, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to get controller runtime configuration, err: %s", err)
 	}
+
+	config, err := LoadConfiguration(ctx, opts.Config, conf)
+	if err != nil {
+		return nil, fmt.Errorf("unable to load configuration, config: %s, err: %w", opts.Config, err)
+	}
+
+	secProvider, err := security.New(ctx, security.Options{
+		SentryAddress:           config.SentryAddress,
+		ControlPlaneTrustDomain: config.ControlPlaneTrustDomain,
+		ControlPlaneNamespace:   security.CurrentNamespace(),
+		TrustAnchorsFile:        opts.TrustAnchorsFile,
+		AppID:                   "dapr-operator",
+		// mTLS is always enabled for the operator.
+		MTLSEnabled:    true,
+		WriteSVIDToDir: ptr.Of(filepath.Join(os.TempDir(), "/var/run/secrets/dapr.io/webhook/tls")),
```

Comment:
Should this use a process-specific tempdir? https://pkg.go.dev/os#MkdirTemp

Reply:
I _think_ we do actually want to have a deterministic path for the SVID as this seems more consistent with other var run secrets. IIRC, the reason we are using temp here is that the operator process doesn't have permissions in the container to create a dir under `/var/run`.